recipe/conda_build_config.yaml

```
@@ -1,97 +1,16 @@
-c_stdlib_version:  # [osx and x86_64]
+# See following link for c_stdlib sysroot constraint addition
+# https://github.com/conda-forge/pytorch-cpu-feedstock/pull/293#issuecomment-2503611320
+# 2024/12 hmaarrfk's summary:
+#   The medium term solution is to add such a constraint to libcufile
+#   The long term solution is to add such a constraint to all packages
+#   that depend on a specific sysroot at building.
+c_stdlib_version:  # [unix]
```

is this syntax correct? shouldn't it be 
```suggestion
c_stdlib_version:  # [(linux and aarch64) or (osx and x86_64)]
```

---

It's fine. It's just an empty spec in the other cases, which is a no-op.

---

this is new syntax to me...... I would think that it might "delete" things....

---

You can try to rerender the PR (I did before pushing - no change). It doesn't change the stdlib, because the merge logic for the various different configs (global & local) ignores empty values